hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMDBCheckpointServlet.java

```diff
@@ -380,18 +408,24 @@ private Set<Path> getSnapshotDirs(DBCheckpoint checkpoint, boolean waitForDir)
         iterator = checkpointMetadataManager
         .getSnapshotInfoTable().iterator()) {
 
-      // For each entry, wait for corresponding directory.
+      // For each entry, wait for corresponding directory if the key is
```

Comment:
Are we assuming that whatever is flushed to disk must have a snapshot directory?